### PR TITLE
message displayed when user do login, solves issue #24

### PR DESCRIPTION
**Pull request by amit6663**

### CODE REVIEW

**Comment by amit6663** [comment]
@rachittt @robustTechie pls review

File: mifos-passcode/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -36,4 +36,5 @@
     <string name="authentication_cancelled">Authentication Cancelled</string>
     <string name="fingerprint">Fingerprint</string>
     <string name="FingerprintEnableMessage">Do you want to enable Fingerprint Authentication?</string>
+    <string name="LSuccess">Login Successfully</string>
```
Comment (rachittt) [inline mifos-passcode/src/main/res/values/strings.xml]:
```suggestion
    <string name="login_successfully">Login Successfully</string>
```

File: mifos-passcode/src/main/res/layout/activity_pass_code.xml [hunk 2]
```diff
@@ -44,6 +44,15 @@
...
 
+            <TextView
+                android:id="@+id/tv_waiting"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="@dimen/default_margin"
+                android:gravity="center|start"
+                android:textSize="@dimen/text_large" />
+
+
```
Comment (rachittt) [inline mifos-passcode/src/main/res/layout/activity_pass_code.xml]:
```suggestion
```

File: mifos-passcode/src/main/java/com/mifos/mobile/passcode/MifosPassCodeActivity.java [hunk 3]
```diff
@@ -58,7 +64,7 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_pass_code);
-
+      tv_timer=findViewById(R.id.tv_waiting);
```
Comment (rachittt) [inline mifos-passcode/src/main/java/com/mifos/mobile/passcode/MifosPassCodeActivity.java]:
```suggestion
         tv_timer=findViewById(R.id.tv_waiting);
```

File: build.gradle [hunk 4]
```diff
@@ -9,12 +9,16 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.2'
+        classpath 'com.android.tools.build:gradle:3.5.3'
```
Comment (rachittt) [inline build.gradle]: please remove this change

File: build.gradle [hunk 5]
```diff
@@ -9,12 +9,16 @@ buildscript {
...
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3'
         classpath 'com.github.dcendents:android-maven-gradle-plugin:2.1'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+
```
Comment (rachittt) [inline build.gradle]:
```suggestion
```

File: build.gradle [hunk 6]
```diff
@@ -9,12 +9,16 @@ buildscript {
...
+        classpath 'com.mifos.mobile:mifos-passcode:1.0.0'
+
```
Comment (rachittt) [inline build.gradle]:
```suggestion
```

File: build.gradle [hunk 7]
```diff
@@ -9,12 +9,16 @@ buildscript {
...
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
+
```
Comment (rachittt) [inline build.gradle]:
```suggestion
```

File: gradle/wrapper/gradle-wrapper.properties [hunk 8]
```diff
@@ -1,6 +1,6 @@
-#Fri Jan 19 13:52:20 IST 2018
+#Fri Mar 20 11:34:39 IST 2020
```
Comment (rachittt) [inline gradle/wrapper/gradle-wrapper.properties]: why did you make this change?

File: gradle/wrapper/gradle-wrapper.properties [hunk 9]
```diff
@@ -1,6 +1,6 @@
...
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```
Comment (rachittt) [inline gradle/wrapper/gradle-wrapper.properties]: Again, I don't think this is necessary in order to solve this issue.

Comment (amit6663) [reply gradle/wrapper/gradle-wrapper.properties]: sir @rachittt pls give me some more description, I'm not getting what u say.

Comment (rachittt) [reply gradle/wrapper/gradle-wrapper.properties]: can you explain why updating the version of gradle necessary for solving this issue?

Comment (amit6663) [reply gradle/wrapper/gradle-wrapper.properties]: gradle was updated itself.

**Pull request closed without merging**